corehq/apps/reports_core/static/reports_core/js/maps.js

```diff
@@ -37,30 +52,30 @@ var maps = (function() {
             });
             var table = '<table class="table table-bordered"><%= rows %></table>';
             var template = _.template(table)({rows: rows.join('')});
-            fn.template = _.template(template);
+            privates.template = _.template(template);
```

Comment:
I don't think you've changed this, but is it true that the first call to `render(config, ...)` causes the module to be initialized, setting up `private.template` using `config.columns`? Subsequent calls to `render(config, ...)` with different `config.columns` will use the template constructed on the first call (old `config.columns`). In other words, this `private` member retains state that affects subsequent rendering. Does that matter?